packages/wonder-blocks-form/src/components/__tests__/text-area.test.tsx

```diff
@@ -1519,10 +1560,11 @@ describe("TextArea", () => {
                         />,
                         defaultOptions,
                     );
-
-                    // Act
                     const field = screen.getByRole("textbox");
                     await userEvent.type(field, "test");
+                    handleValidate.mockReset(); // Reset mock before leaving the field
```

Comment:
Since we now clear the error state onChange when `instantValidation=false` (see `use-field-validation.ts` changes), `onValidate` is called each time a user types a character. We reset the mocks now before the action so we can more clearly check the `onValidate` calls triggered by the action